src/renderers/dom/ReactDOM.js

```diff
@@ -133,6 +133,13 @@ if (__DEV__) {
       }
     }
   }
+
+  // Check if using Polymer and we are not using native shadow
+  var shadyDom = window.Polymer !== undefined && !window.Polymer.useNativeShadow;
```

Comment:
I would prefer we check `!= null` rather than `!== undefined` here in the case where there is a Polymer variable set to null so member access doesn't throw later. I know this is unlikely but still.